docs/getting-started/organization-settings.md

```diff
@@ -23,7 +23,30 @@ Use the audit logs to monitor access to your storage. The Audit logs dialog lets
 
 Set your organization's password policy for credentials. This will not change existing passwords, but changes to passwords (either ones generated automatically or manually) must adhere to the organization's current password policy.
 
-## Inbound network rules
+## Network
+
+### Domains
+
+Custom domains allow you to customize and brand your SFTP To Go SFTP endpoints for both SFTP and web portal access.
+
+To add a new custom domain:
+
+1. Click **Add domain**.
+2. Enter the fully qualified domain you'd like to use as an endpoint. Note that root domains cannot be used (i.e., use files.example.io instead of example.io).
+3. Select the type of endpoint that the domain will be used for:
+    * SFTP/FTPS - For use with FTP clients. Requires adding a CNAME pointing to the endpoint.
+    * Web portal - Accessible with any web browser. Requires adding a CNAME pointing to the endpoint. TLS certificates are managed by SFTP To Go.
+4. Click **Add domain** to return to the list.
+5. Wait for instructions on what records to add in your DNS manager. Either apply them yourself or send them to your network manager.
+6. It may take some time for DNS records to apply. You can wait for the state of your domain to show as "Verified" or click the ... menu button to manually refresh the state.
+7. Once the domain has been verified, you can click the ... menu button to set it as the default domain, which will replace the default domain with the newly set up domain in the SFTP To Go admin dashboard.
+
```

Comment:
@saggineumann I think we should add the table we have here (https://github.com/crazyantlabs/sftptogo-docs/pull/164/files#diff-2882505e068f0a6c0d51657d31d153733bc59c664ce932c64faedba832160e59R15) to the list of domain providers and instructions on setting up DNS records.

Reply:
sure, will do.